apps/settings/js/modules/panel_cache.js

```diff
@@ -32,6 +32,7 @@ define(function(require) {
                        'style/simcard.css',
                        'style/updates.css',
                        'style/downloads.css',
+                       'style/improve_browser_os.css',
```

Comment:
You may see they are not `all` panels (we have 70+ panels) but those who have many styles, or a portion of panel that maintained by other team... So I'd still perfer put single style sheet into settings.css